bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/WZGToLNu2jG_4f_NLO/WZGToLNu2jG_4f_NLO_proc_card.dat

```diff
@@ -0,0 +1,18 @@
+set loop_optimized_output True
+set group_subprocesses Auto
+set ignore_six_quark_processes False
+set loop_color_flows False
+set gauge unitary
+set complex_mass_scheme False
+set max_npoint_for_channel 0
+import model loop_sm
+define l+ = e+ mu+
+define l- = e- mu-
+define vl = ve vm vt
+define vl~ = ve~ vm~ vt~
+define p = g u c d s u~ c~ d~ s~
+define j = g u c d s u~ c~ d~ s~
+define lep = e+ mu+ ta+ e- mu- ta-
+define nu = ve vm vt ve~ vm~ vt~
+generate p p > lep nu z a [QCD] @0
```

Comment:
I may overlook this line previously, you are using "LNu2jG" which means you specify the Z>2j, maybe you could modify the name or decay the Z boson here to make them more consistent

Reply:
Done!